ppr-api/src/ppr_api/models/registration.py

```diff
@@ -36,6 +36,53 @@
 FINANCING_PATH = '/ppr/api/v1/financing-statements/'
 
 
+class CrownChargeTypes(Enum):
```

Comment:
```suggestion
class CrownChargeTypes(str, Enum):
```
makes it comparable without having to to do ```enum.value```

Reply:
Suggested change made.